### ISSUE TITLE
Meeting IA updates designed

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/action#2926]
- This is a stretch goal for the async milestone:
  - The meeting UI was made with sync + laptops in mind, e.g. we don’t need lots of avatars breaking the mobile layout if the meeting is async (just a symptom of the IA problem)
  - The main hunch right now is that async UX may cause an increase in mobile usage (need to validate, test hypothesis, as we go)
  - The second part of that is that mobile may help make the product sticky aka attachment
  - Also, including minimal mobile-first-then-friendly design concepts in future design explorations for features, targets, etc. help us think about essential, primary actions, and make room for a feature-rich product

#### Unblocks
- CTAs in top bar #2868 
- Video/avatar UX
- Can share common patterns in other spaces like dashboard, settings, etc. #2519
- Potentially better patterns for Group, Discuss (including thread) etc.

### Acceptance Criteria
- Level 3 sketches of IA concepts
- Level 2 concepts and prototypes of IA concepts
- For:
  - Team drawer
  - Left nav drawer
  - Top bar
  - Bottom bar
  - Phase IA (patterns like #2905 and #2908)
  - Input IA
  - Avatars including video/audio
- Decisions made about patterns to prioritize
- Implementation issues written

### Estimated Effort
- 12 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by ackernaut** [comment]
Closing this in favor of newer, smaller issues
